<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionMenuFrame</class>
 <widget class="QFrame" name="ConnectionMenuFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="LABEL_BT_DEVICE_ICON">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border: 1px solid rgb(0,0,0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_BT_DEVICE_NAME">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_BT_DEVICE_MAC_ADDRESS">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_UNPAIR">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>220</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>REMOVE</string>
   </property>
  </widget>
  <widget class="QListWidget" name="LISTWIDGET_PAIRED_DEVICE">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>671</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_RENAME_BT_DEVICE">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>40</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>RENAME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_UNPAIR_ALL">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>300</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>REMOVE ALL</string>
   </property>
  </widget>
  <widget class="KeyboardFrame" name="keyboardFrame">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>38</y>
     <width>920</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyboardFrame</class>
   <extends>QFrame</extends>
   <header>Keyboard/KeyboardFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
